app/webpacker/components/Tickets/TicketWorkbenches/EditPersonTicketWorkbench.jsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
+import { Message } from 'semantic-ui-react';
 import EditPersonForm from '../../Panel/pages/EditPersonPage/EditPersonForm';
 import useSaveAction from '../../../lib/hooks/useSaveAction';
 import { actionUrls } from '../../../lib/requests/routes.js.erb';
 import Loading from '../../Requests/Loading';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import Errored from '../../Requests/Errored';
+
+function EditPersonValidations({ ticketDetails }) {
+  const { ticket } = ticketDetails;
+  const {
+    data: validators, loading, error,
+  } = useLoadedData(actionUrls.tickets.editPersonValidators(ticket.id));
+
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
+
+  return validators.dob.map((validator) => (
+    <Message warning>{validator.message}</Message>));
```

Comment:
Nit: There should be a line-break before the last `)`

Reply:
Oops, thanks for pointing. Fixed.